README.md

```diff
@@ -17,9 +17,9 @@ This modules provides integration with the [InfluxDB](https://influxdata.com/) d
 
 ```xml
 <dependency>
-  <groupId>com.github.miwurster</groupId>
+  <groupId>com.github.mbigger</groupId>
```

Comment:
Why changed?

Reply:
I tyr to upload to my maven repo, forget to mofidy it back.